plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/roi_pooling.cpp

```diff
@@ -30,23 +29,27 @@ const std::vector<std::vector<size_t>> pooledShapes_bilinear = {
 const std::vector<std::vector<size_t>> coordShapes = {
     {1, 5},
     {3, 5},
-    {5, 5},
 };
 
 const std::vector<InferenceEngine::Precision> netPRCs = {
-    InferenceEngine::Precision::FP16,
```

Comment:
Are the FP16 tests broken or are you just trying to reduce the testing size? Knowing whether FP16 is working is important, and if FP16 is working we should retain the FP16 tests even if they are slow. (If FP16 is broken we can omit FP16 tests like you're doing here, but we will need to track the work of eventually fixing them.)

Reply:
yes, FP16 tests are broken with `bilinear` mode. for now, I just comment it, but we find that FP16 issue shows in many other ops, we thought it maybe a common issue, and will fix it in a separate PR.

Reply:
Yes, makes sense.